static/app/utils/discover/eventView.tsx

```diff
@@ -430,7 +430,8 @@ class EventView {
         },
         'environment'
       ),
-      yAxis: saved.yAxis,
+      // Workaround to only use the first yAxis since eventView yAxis doesn't accept string[]
+      yAxis: Array.isArray(saved.yAxis) ? saved.yAxis[0] : saved.yAxis,
```

Comment:
Talked with @edwardgou-sentry in slack and he mentioned it's getting messy to type this to `string[]`. Not sure what the implications of this are. For example, in results.tsx, we check `eventView.getYAxis()` to see if chart parameters have changed etc... is that code still valid?

Reply:
Yup unfortunately a lot of tsx code areas (outside of discover and dashboards as well) depend on `EventView.yAxis` belonging to type `string | undefined` at the moment.

It's probably possible to refactor `EventView.yAxis` to `string[] | undefined` but that would blow up this pr a little. I'm thinking we may be able to tackle this in a separate pr?

Also, I've gone back and taken a look at references to `EventView.yAxis`, and I think the `results.tsx` example is the only outstanding one that needs to be update for now. I've updated this to check the full yAxis array from the `SavedQuery`!

Reply:
@edwardgou-sentry Can you add a test case for this in `js/spec/utils/discover/eventView.spec.jsx`? Ensuring passing an array of y-axes will only pick the first item of the array for `EventView.fromSavedQuery()`.

Reply:
Yup, added!